editor/plugins/skeleton_3d_editor_plugin.cpp

```diff
@@ -183,16 +269,17 @@ void BoneTransformEditor::_property_keyed(const String &p_path, bool p_advance)
 	}
 }
 
-void BoneTransformEditor::_update_properties() {
+void BonePropertiesEditor::_update_properties() {
 	if (!skeleton) {
 		return;
 	}
 	int selected = Skeleton3DEditor::get_singleton()->get_selected_bone();
 	List<PropertyInfo> props;
+	HashMap<StringName, bool> meta_seen;
```

Comment:
```suggestion
	HashSet<StringName> meta_seen;
```
No need for a map where you never use the value

Reply:
Thanks, that's better